packages/code-du-travail-frontend/src/common/Answer.js

```diff
@@ -14,11 +16,27 @@ const BigError = ({ children }) => (
   </Container>
 );
 
+const BackToResultsLink = ({ search }) => {
+  if (!search) return null;
+
+  return (
+    <Container>
+      <Link route="index" params={{ q: search }}>
+        <a>{"< Retour aux résultats"}</a>
+      </Link>
+    </Container>
+  );
+};
+
 class Answer extends React.Component {
   state = {
-    modalVisible: false
+    modalVisible: false,
+    isMounted: false
```

Comment:
`isMounted` What is it for ?

Reply:
Bordel j'ai fait ça comme un chiffonier. A rien, il sert à rien !